modules/low-watermark/src/main/java/org/apache/ignite/internal/lowwatermark/configuration/LowWatermarkConfigurationSchema.java

```diff
@@ -24,9 +24,7 @@
 import org.apache.ignite.configuration.annotation.Value;
 import org.apache.ignite.configuration.validation.Range;
 
-/**
- * Low watermark configuration schema.
- */
+/** Low watermark configuration schema. */
```

Comment:
:(

Reply:
This doesn’t bother me, if it bothers you, you can bring it up for discussion yourself and create a rule or something like that.

Reply:
> This doesn’t bother me

Why do you keep updating the code then?

Reply:
I just like this style.

Reply:
It's not your personal project